cms/tests/test_po.py

```diff
@@ -33,10 +35,13 @@ def compile_messages():
         for dirpath, dirnames, filenames in os.walk(basedir):
             for f in filenames:
                 if f.endswith('.po'):
-                    fn = os.path.join(dirpath, f)
-                    if has_bom(fn):
-                        raise CommandError("The %s file has a BOM (Byte Order Mark). Django only supports .po files encoded in UTF-8 and without any BOM." % fn)
-                    pf = os.path.splitext(fn)[0]
+                    if DJANGO_2_2 or DJANGO_3_0 or DJANGO_3_1:
+                        pfn = os.path.join(dirpath, f)
+                    elif DJANGO_3_2:
```

Comment:
With "elif DJANGO_3_2: instead of "else:", what happens when django 3.3, etc. is released? As I understand it, there is a matrix of (ptyhon /) django resleases intended for a specific version of Django CMS.  So I would have used "else:" and based on that matrix, adapt the code later for the next version of (python /) django.

Reply:
Now, when I think more deeply. I think you are right. Using an else, makes it easily future proof because if there is a change, we can just add a new condition.

Reply:
I will make these changes and commit later tonight.

Reply:
@pcolmat please check my latest commit.